### ISSUE TITLE
Lower bound of `Number` whisper component has no effect

### ISSUE THREAD

**Issue by hans-hsieh-asus** [open-olive/loop-development-kit#192]
When using the `Number` whisper component, it looks like the `min` doesn't have an effect. E.g. 
```javascript
await whisperService.create({
      label: 'Number input',
      components: [
        {
          type: whisperService.WhisperComponentType.Markdown,
          body: 'This is a "number input" component without box.',
        },
        {
          type: whisperService.WhisperComponentType.Number,
          label: 'range [0, 10], step 2',
          value: 0,
          min: 0,
          max: 10,
          step: 2,
          tooltip: 'a tooltip',
          onChange: (newValue) => console.log(`New number: ${newValue}`),
        },
      ],
      onClose: () => {},
    });
```
The `max` has an effect though.
LDK version: 3.0.0-beta.3
OliveHelps version: 0.17.3

**Comment by richseviora** [comment]
Thanks for reporting this Hans! SIDE-2319 created for tracking this. We'll look at this and get back to you soon.

**Comment by richseviora** [comment]
It looks like this issue only manifests itself when the `min` value is set to `0`. If I set it to `-2` the value is passed through to the UI and respected by the UI.

**Comment by richseviora** [comment]
This issue has been fixed in v18.10, which was released today! Thanks again for reporting the bug 🙂
